dist/well-known.js

```diff
@@ -125,11 +127,15 @@ return Promise.all([
       return data;
     });
   }),
-  parseResponse('/.well-known/security.txt', r => {
+  parseResponseWithRedirects('/.well-known/security.txt', r => {
+    let data = {
+      status: r.status,
+      redirected: r.redirected,
+      url: r.url
+    };
```

Comment:
I like the idea of having these information. It will help to provide metrics to verify the `canonical` with the redirected URL. Plus the signed data. That addresses some of the stuff mentioned in https://www.rfc-editor.org/rfc/rfc9116#section-5.1

Reply:
Yeah just note that `status` won't be 3xx in this case, since it refers to the status of the final page, so be sure to use the `redirected` field instead.